packages/foreman/katello-selinux/katello-selinux.spec

```
@@ -127,7 +127,66 @@ fi
 %{_mandir}/man8/%{name}-disable.8.gz
 %{_mandir}/man8/%{name}-relabel.8.gz
 
+%package -n crane-selinux
+Summary: SELinux policy module for Crane
+Group:   System Environment/Base
+
+Requires:           selinux-policy >= %{selinux_policy_ver}
+Requires(post):     /usr/sbin/semodule
+Requires(post):     /sbin/restorecon
+Requires(post):     /usr/sbin/setsebool
+Requires(post):     /usr/sbin/selinuxenabled
+Requires(post):     /usr/sbin/semanage
+Requires(post):     selinux-policy-targeted
+Requires(postun):   /usr/sbin/semodule
+Requires(postun):   /sbin/restorecon
+
+%if 0%{?rhel} == 7
+Requires(post):     policycoreutils-python
+%else
+Requires(post):     policycoreutils-python-utils
+%endif
+
+%description -n crane-selinux
+SELinux policy module for Crane
+
+%post -n crane-selinux
+if /usr/sbin/selinuxenabled; then
+    # install and upgrade
+    %{_sbindir}/crane-selinux-enable
+fi
+
+%posttrans -n crane-selinux
+if /usr/sbin/selinuxenabled; then
+    # install and upgrade
+    %{_sbindir}/crane-selinux-relabel
+fi
+
+%preun -n crane-selinux
+if /usr/sbin/selinuxenabled; then
+    # uninstall only
+    if [ $1 -eq 0 ]; then
+        %{_sbindir}/crane-selinux-disable
+    fi
+    # upgrade and uninstall
+    %{_sbindir}/crane-selinux-relabel
+fi
+
+%files -n crane-selinux
+%doc LICENSE crane.fc crane.if crane.te
```

Also not optimal in the main package, but I like to use `%license`
```suggestion
%license LICENSE
%doc crane.fc crane.if crane.te
```

---

:+1: fixed it in both places